about.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>About</title>
+    <link rel="stylesheet" href="style-ab.css" />
+  </head>
+  <body>
+    <!-- nav mobile -->
+    <header>
+      <nav class="navbar">
+        <a href="./index.html" class="toplogo"><img src="./assets/Images/chililogo.png" alt="band logo"></a>
+        <ul class="nav-menu">
+          <li class="nav-item"><a href="./index.html" class="nav-link">Home</a></li>
+          <li class="nav-item"><a href="#program" class="nav-link">Program</a></li>
+          <li class="nav-item"><a href="#tickets" class="nav-link">Tickets</a></li>
+          <li class="nav-item"><a href="#sponsors" class="nav-link">Sponsor</a></li>
+          <li class="nav-item"><a href="#" class="nav-link">News</a></li>
+                    <li class="nav-item">
+            <a href="#" class="nav-link funky">Get Funky!</a>
+          </li>
+        </ul>
+        <div class="hamburger">
+          <span class="bar"></span>
+          <span class="bar"></span>
+          <span class="bar"></span>
+        </div>
+      </nav>
+    </header>
+    <!-- Title Page -->
+    <section class="landing">
+      <p class="above-header">Join us for the global premiere!</p>
+      <img src="./assets/Images/LOGO.png" alt="unlimited love" class="logo">
+      <h1 class="header1">by the Red Hot Chili Peppers</h1>
+
+      <p class="above-h2 second">
+        The concert that will bring together every fan, casual or hard-coded, to share an unique moment in music history. 
+Never before has an event like this has been made in the history of Rock!
+      </p>
+      <p class="above-h2 second">
+        Want to join us? Grab a ticket at the homepage or send us a few lines @
+      </p>
+      <a href="mailto:unlimitedlove@rhcp.com">unlimitedlove@rhcp.com</a>
+
+      </section>
+
+      <div class="newalbum">
+      <h2 class="second">THE RED HOT <br> CHILI PEPPERS</h2>
+      <p class="second">Can't join us? Listen to the album on Spotify!</p>
+      <div class="listen">
+        <a href="https://open.spotify.com/album/2ITVvrNiINKRiW7wA3w6w6"><img src="./assets/Images/listentoalbum.jpg" alt="album art"></a>
+      </div>
+    </div>
```

Comment:
- [ ] Kindly you can change the content of the original design, but you have to stick to the layout.
- [ ] Kindly use the `section` tag instead of `div`, it's more semantic.
- [ ] Kindly replace the big image with your logo, or a small image to match the original layout.

Your design
![image](https://user-images.githubusercontent.com/35535130/163291156-92368f71-bfe7-4678-a6f4-0f32ae859fa4.png)



Original design
![image](https://user-images.githubusercontent.com/35535130/163291090-9c123335-03fc-42c0-b09c-dfa195454c87.png)

Reply:
I solved this issues as well, thank you!